test/examples/runtests.jl

```diff
@@ -49,7 +49,7 @@ if "spheres" in cases
         np=numprocs,
         rtol=reltol,
         atol=abstol,
-        excluded_columns=["Maximum", "Minimum"]
```

Comment:
The number of elements ought to be large enough that the mean should be stable if the norm is stable, so this is a bit surprising. Roughly how big were the misses that meant `Mean` needed to be added?

Reply:
Here's an example from the `cavity_impedance.json` case:

```
Regression test error at row 1, column Mean: 0.0001699570272 ≉ 0.0001700702508
```

Very small errors, but there are only 12 elements in the mesh (96 after refinement) and the eigenvalue solve is particularly sensitive to very small differences between repeated runs. I will just exclude `"Mean"` from the cavity examples instead of adding it to the exclusion list for all examples. Does that sound OK?

Reply:
Resolved in [8c7b4b6](https://github.com/awslabs/palace/pull/68/commits/8c7b4b61b721c84d30904f93945c2a2d3ef33a6d) by just tightening the linear solver tolerance.

Reply:
Closing the loop here:

This has nothing to do with linear solver tolerance/mesh resolution. The cylindrical cavity resonator modes have a rotational invariance; different eigenmode solvers may converge to a mode which has a rotation about the cylinder center axis by some nonzero angle. In this case, the global norm of the error esimate will still be the same, but the elementwise contributions will not agree. Thus, it's correct to ignore the `"Mean"` field (along with `"Min"` and `"Max"`), and we were probably just getting lucky earlier with the tests passing.

An alternative approach would be to simulate only half the cylinder, with a symmetry plane (PEC or PMC depending on the modes sought). This removes the rotational symmetry and is an option tested in [cbcb4dd](https://github.com/awslabs/palace/pull/68/commits/cbcb4ddb5d0040858bfe989bd54caa1c593258f0).